app/controllers/admin/applicants_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Admin::ApplicantsController < ApplicationController
+  def index
+    @applicants = Applicant.where(application_status: "Pending")
+  end
+  
+  def show
+    @applicant = Applicant.find(params[:id])
+    @applicant_pets = @applicant.pets
+  end
+
+  def approve_pet
```

Comment:
I think both your `approve_pet` and `reject_pet` methods could potentially live in a `update` action with some control flow `if/else`. However, I can also see how this could help with readability for other devs.